src/nrncvode/netcvode.cpp

```diff
@@ -3065,8 +3064,7 @@ void PreSyn::deliver(double tt, NetCvode* ns, NrnThread* nt) {
             TQItem* q = ns->p[i].tq_->least();
             Cvode* cv = (Cvode*) q->data_;
             if (tt < cv->t_) {
-                int err = NVI_SUCCESS;
-                err = cv->handle_step(nrn_ensure_model_data_are_sorted(), ns, tt);
+                cv->handle_step(nrn_ensure_model_data_are_sorted(), ns, tt);
```

Comment:
handle_step does return an error status. Should we print a warning if not NVI_SUCCESS?

Reply:
Could you write me a message for that?

Reply:
done